src/System.Runtime.TimeZoneData/System.Runtime.TimeZoneData.csproj

```
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+      <TimeZoneDataVersion>2020a</TimeZoneDataVersion>
+      <TimeZoneSource>https://data.iana.org/time-zones/tzdb</TimeZoneSource>
```

nit: could be simplified to this so we don't need to append `$(TimeZoneDataVersion)` later on:

```suggestion
      <TimeZoneSource>https://data.iana.org/time-zones/tzdb-$(TimeZoneDataVersion)</TimeZoneSource>
```